src/Resources/Publications.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Zzz\ShopifyGraphql\Resources;
+
+use Saloon\Http\Connector;
+use Saloon\Traits\Plugins\AlwaysThrowOnErrors;
+use Zzz\ShopifyGraphql\Requests\Products\GetPublications;
+use Zzz\ShopifyGraphql\Requests\Products\PublishProduct;
+use Zzz\ShopifyGraphql\Responses\Publications\PublicationsResponse;
+use Zzz\ShopifyGraphql\Trait\ValidateGraphQlResponse;
+
+class Publications
+{
+    use AlwaysThrowOnErrors;
+    use ValidateGraphQlResponse;
+
+    public function __construct(protected Connector $api)
+    {
+    }
+
+    public function getPublications(): PublicationsResponse
+    {
+        $response = $this->api->send(new GetPublications());
+
+        $this->validate($response);
+
+        return new PublicationsResponse($response->json('data.publications') ?? []);
+    }
+
+    public function publishProduct(string $productId, string $publicationId): void
+    {
+        $response = $this->api->send(
+            new PublishProduct(
+                productId: $productId,
+                attributes: [
+                    'publicationId' => $publicationId
+                ]
+            )
+        );
+
+        $this->validate($response);
```

Comment:
Just a question as why you decided to not return anything here?

Reply:
@tonning  Well, I thought about that for like 10 minutes 😂 and was like; I didn't see any use case for the returned data, especially because validation would always work as well incase something is not right, but after that, nothing else to do wit the data returned. 

But anyways; might not be the best call. What do you think?